src/data/providers/initPoolDataProvider.ts

```diff
@@ -49,22 +50,35 @@ export class InitPoolDataProvider {
             address: poolAddress,
             publicClient: this.client,
         });
+
+        const poolTokens = amounts
+            .map(({ address, decimals }, index) => ({
+                address: address.toLowerCase() as Address,
+                decimals,
+                index: index + 1,
+            }));
+
+        const poolTokensWithBpt = sortTokensByAddress([
+            {
+                address: poolAddress.toLowerCase() as Address,
+                decimals: 18,
+                index: 0,
```

Comment:
Is this truly always zero? I was under the impression it would be index of its position on the sorted array.